### ISSUE TITLE
calling setByValue() inside onSelect causes dropdown to stay open

### ISSUE THREAD

**Issue by tar-viturawong-sociomantic** [sociomantic-tsunami/flounder#103]
**Scenario:** Suppose we have two dropdowns, where manual selection of the first leads to programmatic selection of the second. 

In code:

```javascript
document.body.innerHTML = `
<div class="js-flounder1"></div>
<div class="js-flounder2"></div>
`;

var flounder1 = new Flounder( document.querySelector( '.js-flounder1' ), 
{
    data : 
    [
        { text : 'apple', value : '0' },
        { text : 'pear',  value : '1' }
    ],

    onSelect : () => flounder2.setByValue( '10' )
} );

var flounder2 = new Flounder( document.querySelector( '.js-flounder2' ),
{
    data : [ '1', '10', '100' ]
} );

// now, select something in the first dropdown
```

**Currently**
The first dropdown is stuck in the open state.

**Wanted**
Close the first dropdown.

@mousemke In case this is intended behavior and that I'm actually mis-/ab-using `setByValue()`, could you please provide an alternative way of achieving the scenario above? Thanks much!

**Comment by mousemke** [comment]
i'll check it out

**Comment by mousemke** [comment]
do you need this in 0.8.X, or can this go in 1.0.0?

**Comment by tar-viturawong-sociomantic** [comment]
`0.8.X` would be awesome! 🙏

**Comment by mousemke** [comment]
it appears to actually be opening and then closing, as opposed to staying open

**Comment by tar-viturawong-sociomantic** [comment]
Hmm it stays on open for me when I run the code above in the console of http://flounderjs.com/demo/ (Chrome 51.0.2704.103). @kevin-krug-sociomantic are you able to reproduce this?

**Comment by mousemke** [comment]
you cant see it (happening quickly), but that was it.  events bubbling and triggering `toggleList` twice.  fix ready, up later today (`0.8.5`)

**Comment by tar-viturawong-sociomantic** [comment]
Ah yes now I see what you meant by _actually [be] opening and then closing_; that's what I saw when debugging too but the net effect was that it looked like it stayed open (as unpaused synchronous execution doesn't update the DOM). Anyway, thanks much for the quick fix @mousemke!

**Comment by mousemke** [comment]
fixed in 0.8.5 

https://github.com/sociomantic-tsunami/flounder/releases/tag/0.8.5
